src/vmeasure.jl

```diff
@@ -0,0 +1,70 @@
+"""V-measure of contingency table"""
+function _vmeasure(A::Matrix{Int}; β = 1.0)
+    C, K = size(A)
+    N = sum(A)
+
+    # Homogeneity
+    hck = 0.0
+    for k in 1:K
+        d = sum(A[:,k])
+        for c in 1:C
+            if A[c,k] != 0 && d != 0
+                hck += log(A[c,k]/d) * A[c,k]/N
+            end
+        end
+    end
+    hck = -hck
+
+    hc = 0.0
+    for c in 1:C
```

Comment:
If we can afford allocation, it could be simplified (up to the scale?) into `entropy(sum(A, 1))`.
`A[c, :]` allocates anyway, should be changed into `view(A, c, :)`.